fixtures/geojson-change-projection.yaml

```diff
@@ -10,7 +10,7 @@ App\Entity\DataFlow:
     geojson_change-projection_data_flow:
         name: 'GeoJSON: Change projection'
         dataSource: '@geojson_change-projection_data_source'
-        frequency: 60
+        schedule: 1 * * * *
```

Comment:
Just for fun, we could let one of the `schedule` values in a single fixture be something other than `1 * * * *`.

Reply:
Good idea, I will change that for some of the flows.